src/main/resources/config.yml

```diff
@@ -1,4 +1,4 @@
 options:
   auto-update: true
-  need-wrench-to-break-machines: true
-  wrench-failure-chance: 0.3
+  need-wrench-to-break-machines: false
```

Comment:
this true
but make it false for new servers

Reply:
? this is for new servers

Reply:
both existing and new servers will have it default off, i had it on true by mistake

Reply:
Should be true for new servers. Just false for existing.